supported_platforms.md

```diff
@@ -20,6 +20,8 @@ Nightly test platform details:
 .NET versions tested on
 - .NET 7.0
 - .NET 6.0
+- .NET Core 3.1
+- .NET Framework 4.6.1 (only IoT Hub SDKs)
```

Comment:
Why only Hub SDKs? Is net461 not .netstandard2.0 compliant?

Reply:
net461 isn't fully compliant with ns2.0. For this reason I think we should test against net472 instead (which is what we have been doing). What do you think? @drwill-ms 

From https://learn.microsoft.com/en-us/dotnet/standard/net-standard?tabs=net-standard-2-0#select-net-standard-version:
The versions listed here represent the rules that NuGet uses to determine whether a given .NET Standard library is applicable. While NuGet considers .NET Framework 4.6.1 as supporting .NET Standard 1.5 through 2.0, there are several issues with consuming .NET Standard libraries that were built for those versions from .NET Framework 4.6.1 projects. For .NET Framework projects that need to use such libraries, we recommend that you upgrade the project to target .NET Framework 4.7.2 or higher.

Reply:
Wow, I wonder why track2 tests with it then.

I'd say stick with 472 for now, but ask the azure sdk team about it.